architecture/verifier/freshness.md

```diff
@@ -0,0 +1,101 @@
+## Freshness
+
+In order to prevent replay attacks, Veraison employs a two-stage
+challenge-response protocol for verification. First, a verification session is
+created, and a nonce for that session is established. The nonce is used as a
+challenge when generating the attestation report/token which is presented in
+the second stage that results in a signed attestation result produced by the
+verifier.
+
+Since the way the nonce is incorporated in the attestation token depends on
+the specifics of the attestation scheme, the core Veraison verifier only
+propagates the session nonce (established in the first stage), along with the
+evidence extracted from the attestation token. There are three options for
+how/where token freshness can be validated:
+
+1. by the attestation scheme's Evidence Handler plugin
+2. by the deployment-specific policy associated with the scheme
+3. left to the Relying Party verifier based on the attestation result (which
```

Comment:
I understand you are using the term Relying Party verifier, however to keep the things simple we can confine the term `verifier` to the scope of Veraison only, if possible!

Reply:
Sure,  I see your point.

Reply:
I don't have particularly strong feelings, and I understand the potential for a slight mental shock in the uninitiated, but I think "RP verifier" is marginally better than just "RP".  Anyway, I leave this to the discretion of the editor ;)